addon/mixins/axes.js

```diff
@@ -16,11 +16,65 @@ export default Ember.Mixin.create({
   graphicHeight: null,
   minXTicks: 3,
   minYTicks: 3,
-  tickSpacing: 50,
-
   minAxisValue: 0,
   maxAxisValue: 0,
 
+  /**
+   * We used to have only one option to set tick spacing for both x and y axes.
+   * We keep this attribute for backward compatibility.
+   * @type {number}
```

Comment:
If it's just a bug with GitHub, that's fine. Just want to make sure it's not because there's something wrong with the code